ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -199,21 +199,22 @@ azure_flush(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session, WT_FILE_SYST
     azure_file_system *azure_fs = reinterpret_cast<azure_file_system *>(file_system);
     WT_FILE_SYSTEM *wtFileSystem = azure_fs->wt_fs;
     auto log = azure_fs->store->log.get();
+    std::string local_file_path = azure_fs->home_dir + "/" + source;
```

Comment:
Is this expected to work on Windows? If so, it'd be better to use the standard library's path manipulation code.

Reply:
The storage sources currently is not expected to work for platforms other than Linux.

Reply:
Fair enough, thanks.